mne/viz/utils.py

```diff
@@ -138,10 +138,14 @@ def _show_browser(show=True, block=True, fig=None, **kwargs):
     if backend == 'matplotlib':
         plt_show(show, block=block, **kwargs)
     else:
+        from matplotlib import rcParams
         from PyQt5.QtWidgets import QApplication
         from .backends._utils import _qt_app_exec
         if show:
             fig.show()
+            if rcParams['figure.raise_window']:
+                fig.activateWindow()
+                fig.raise_()
```

Comment:
Would it make more sense to override the `show` method at the mne-qt-browser backend to `super().show()` then add these lines? I think the less Qt-backend-specific stuff we can have in MNE-Python, the better.

Reply:
(or if you already have a `def show(...)` in there, you can just add it there)

Reply:
Agreed, will do